CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to the AutoGluon Dashboard. Whether it's a bug report, new feature, correction, or additional
+documentation, we greatly value feedback and contributions from our community.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+## Reporting Bugs/Feature Requests
+
+We welcome you to use the GitHub issue tracker to report bugs or suggest features.
+
+When filing an issue, please check [existing open](https://github.com/autogluon/autogluon-dashboard/issues), or [recently closed](https://github.com/autogluon/autogluon-dashboard/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), issues to make sure somebody else hasn't already
+reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
+
+* A reproducible test case or series of steps
+* The version of AutoGluon being used, the version of pytorch
+* Any modifications you've made relevant to the bug
+* Anything unusual about your environment or deployment
+
+Ideally, you can install AutoGluon-Dashboard and its dependencies in a fresh virtualenv to reproduce the bug.
+
+
+## Contributing via Pull Requests
+Code contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
+
+1. You are working against the latest source on the *main* branch.
+2. You check existing open, and recently merged, pull requests to make sure someone else hasn't addressed the problem already.
+3. You open an issue to discuss any significant work - we would hate for your time to be wasted.
+
+To send us a pull request, please:
+
+1. Fork the repository.
+2. Modify the source (see details below); please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
+3. Ensure local tests pass.
```

Comment:
Is passing unit tests enough? Does the contributor need to build the panel to verify if it works?

Reply:
NVM, saw you mentioned that below